dry_rest_permissions/generics.py

```diff
@@ -164,6 +189,17 @@ def _get_action(self, action):
             return_action = 'update'
         return return_action
 
+    def _get_permission(self, model_class, action):
+        """
+        Utility function that creates the permission name and checks if it exists.
+        """
+        app_name = model_class._meta.app_label
+        model_name = model_class._meta.model_name
+
+        codename = f"{action}_{model_name}"
+        exists = Permission.objects.filter(codename=codename, content_type__app_label=app_name).exists()
+        return f'{app_name}.{codename}', exists
```

Comment:
I think a function should only do 1 thing, so you would need one function to create a permission name and another one that check for a given permission on a given app the permission exists